docs/source/depth_estimation.mdx

```diff
@@ -29,13 +29,15 @@ Load the `train` split of the dataset and take a look at an example:
 >>> example = train_dataset[index]
 >>> example
 {'image': <PIL.PngImagePlugin.PngImageFile image mode=RGB size=640x480>,
- 'depth_map': <PIL.PngImagePlugin.PngImageFile image mode=L size=640x480>}
+ 'depth_map': <PIL.TiffImagePlugin.TiffImageFile image mode=F size=640x480>}
 ```
 
 The dataset has two fields:
 
-* `image`: a PIL image object.
-* `depth_map`: depth map of the image.
+* `image`: a PIL PNG image object with `uint8` data type.
+* `depth_map`: a PIL Tiff image object with `float32` data type which is the depth map of the image.
+
+It is mention-worthy that, JPEG/PNG format can only store `uint8` or `uint16` data. As depth map is `float32` data, it can't be stored using PNG/JPEG. However, we can save depth map using TIFF format as it supports a wider range of data types, including `float32` data.
```

Comment:
```suggestion
It is mention-worthy that JPEG/PNG format can only store `uint8` or `uint16` data. As the depth map is `float32` data, it can't be stored using PNG/JPEG. However, we can save the depth map using TIFF format as it supports a wider range of data types, including `float32` data.
```